.travis.yml

```diff
@@ -1,4 +1,7 @@
 
 language: R
-sudo: false
-cache: packages
+sudo: required
```

Comment:
Is sudo actually required by this now? (It's not a problem if it is, but the tests will run faster if not.)